fvtest/porttest/CMakeLists.txt

```diff
@@ -112,21 +112,29 @@ target_compile_options(sltestlib
 
 if(OMR_HOST_OS STREQUAL "aix")
 	omr_add_library(aixbaddep SHARED
-		aixbaddep/sltest.c
+			aixbaddep/sltest.c
 	)
 
 	target_compile_options(aixbaddep
 		PRIVATE
 			${OMR_PLATFORM_SHARED_COMPILE_OPTIONS}
 	)
 	
-	set_target_properties(aixbaddep PROPERTIES LINK_FLAGS "-bI:${CMAKE_CURRENT_SOURCE_DIR}/aixbaddep/dummy.imp" FOLDER fvtest)
+	set_target_properties(aixbaddep
+		PROPERTIES
+			LINK_FLAGS "-bI:${CMAKE_CURRENT_SOURCE_DIR}/aixbaddep/dummy.imp"
+			FOLDER fvtest
+	)
+
 	omr_add_exports(aixbaddep
 		sl_AixDLLMissingDependency_function
 	)
 endif()
 
-set_target_properties(omrporttest sltestlib PROPERTIES FOLDER fvtest)
+set_target_properties(omrporttest sltestlib
+	PROPERTIES
+		FOLDER fvtest
+)
 
 if (NOT OMR_HOST_OS STREQUAL "aix" AND NOT OMR_HOST_OS STREQUAL "zos")
 	add_test(NAME porttest COMMAND omrporttest --gtest_output=xml:${CMAKE_CURRENT_BINARY_DIR}/omrporttest-results.xml)
```

Comment:
you can make this:
```cmake
add_test(
	NAME porttest
	COMMAND omrporttest --gtest_output=xml:${CMAKE_CURRENT_BINARY_DIR}/omrporttest-results.xml
)
```

Same for the test added below...

Reply:
Sounds good, I will reformat the add test.